conjureup/juju.py

```diff
@@ -239,6 +240,10 @@ def add_config(k, v):
         add_config("bootstrap-timeout", app.conjurefile['bootstrap-timeout'])
     if app.conjurefile['bootstrap-to']:
         cmd.extend(["--to", app.conjurefile['bootstrap-to']])
+    if app.conjurefile['model-config']:
```

Comment:
This may not be needed as it would only apply during bootstrap to default models (im guessing just the controller model)